js/richtextbox/shared/RichTextbox.svelte

```diff
@@ -0,0 +1,388 @@
+<script lang="ts">
+	import { createEventDispatcher, tick } from "svelte";
+	import { BlockTitle } from "@gradio/atoms";
+	import { Copy, Check, Plus, Clear, File as FileIcon } from "@gradio/icons";
+	import { fade } from "svelte/transition";
+	import type { SelectData } from "@gradio/utils";
+	import type { FileData } from "@gradio/upload";
+
+	export let value: {
+		text: string | null;
+		files: string[] | FileData[];
+	} = { text: null, files: [] };
+	export let lines: number = 1;
+	export let placeholder: string = "Type here...";
+	export let label: string;
+	export let info: string | undefined = undefined;
+	export let disabled = false;
+	export let show_label: boolean = true;
+	export let container: boolean = true;
+	export let max_lines: number;
+	export let show_copy_button: boolean = false;
+	export let rtl = false;
+	export let autofocus: boolean = false;
+	export let text_align: "left" | "right" | undefined = undefined;
+	export let file_count: string;
+	export let file_types: string[] = [];
+	export let include_file_metadata = true;
+
+	let el: HTMLTextAreaElement | HTMLInputElement;
+	let copied = false;
+	let timer: NodeJS.Timeout;
+	let hidden_upload: HTMLInputElement;
+	let accept_file_types: string | null;
+
+	if (file_types == null) {
+		accept_file_types = null;
+	} else {
+		file_types = file_types.map((x) => {
+			if (x.startsWith(".")) {
+				return x;
+			}
+			return x + "/*";
+		});
+		accept_file_types = file_types.join(", ");
+	}
+
+	$: value, el && lines !== max_lines && resize({ target: el });
+
+	const dispatch = createEventDispatcher<{
+		change: undefined;
+		submit: undefined;
+		blur: undefined;
+		select: SelectData;
+		input: undefined;
```

Comment:
These should also be typed as `never`:
```suggestion
		change: never;
		submit: never;
		blur: never;
		select: SelectData;
		input: never;
```